src/Websites/Websites/Websites.csproj

```diff
@@ -11,10 +11,17 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.8.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Graph.RBAC.Fluent" Version="1.35.0" />
+    <PackageReference Include="Microsoft.Azure.Management.KeyVault" Version="3.1.0" />
```

Comment:
Your powershell module should not depend on others' SDKs, to avoid assembly conflict when 2 modules rely on same dll but different versions.
Instead, use the common SDKs that are shared across all the modules:
- https://github.com/Azure/azure-powershell-common/tree/master/src/Graph.Rbac
- https://github.com/Azure/azure-powershell-common/tree/master/src/KeyVault